Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -667,5 +667,25 @@ public virtual void SceneReady(Scene scene, LoadSceneMode mode)
 
 			}
 		}
-	}
+
+		/// <summary>
+		/// A helper function to retrieve a GameObject by its network id.
+		/// </summary>
+		/// <param name="id">Network id of the gameobject</param>
+        public GameObject GetGameObjectByNetworkId(uint id)
+        {
+            if (Networker == null ) //Only check Networker, as NetworkObjects are always initiliased.
+            {
+                Debug.LogWarning("Networker is null. Check if initiliased");
```

Comment:
Maybe reword this to something like `Networker is null. NetworkManager has not been initialised.` 🤔